src/components/views/elements/Pill.js

```diff
@@ -140,8 +147,17 @@ const Pill = React.createClass({
                 }
             }
                 break;
+            case Pill.TYPE_GROUP_MENTION: {
+                const cli = MatrixClientPeg.get();
+
+                try {
+                    group = await FlairStore.getGroupProfileCached(cli, resourceId);
+                } catch (e) { // if FlairStore failed, rely on js-sdk's store which lacks info
```

Comment:
I'm failing to see *which* info the js-sdk is lacking here. For the pill, we need the group ID, avatar and name; all of which are stored in the js-sdk state, no?